src/frame/sdk/UserApi.ts

```diff
@@ -88,6 +88,12 @@ export const deleteDevice = (deviceId?: number) => {
 export const deleteDeviceBySigningKey = (publicSigningKey: Base64String) =>
     UserApiEndpoints.callUserDeviceDeleteBySigningKey(publicSigningKey).map((r) => r.id);
 
+/**
+* Delete a device from the DB given the public signing key of the device. Uses JWT auth.
+*/
+export const deleteDeviceBySigningKeyWithJwt = (jwtToken: string, publicSigningKey: Base64String) =>
```

Comment:
It seems like this makes more sense out on the parent IronWeb next to `createNewDeviceKeys`. I couldn't find anything that exists on user that doesn't require the user to be logged in.

If we move it out we also can rename to `deleteDeviceBySigningKey`